GitCommands/Git/GitCommandHelpers.cs

```diff
@@ -776,94 +704,48 @@ public static string StartBisectCmd()
 
         public static string ContinueBisectCmd(GitBisectOption bisectOption, params string[] revisions)
         {
-            var bisectCommand = GetBisectCommand(bisectOption);
-            if (revisions.Length == 0)
+            var args = new ArgumentBuilder
             {
-                return bisectCommand;
-            }
+                "bisect",
+                bisectOption,
+                revisions
```

Comment:
revisions.Length==0

Reply:
In this case it's fine because the relevant `Add` method doesn't modify the argument string for null or empty enumerables (nor for null or white-space items they produce).

Reply:
correct. My head got spinning just by reviewing this